### ISSUE TITLE
Ontology annotation relate to different properties

### ISSUE THREAD

**Issue by flekschas** [parklab/refinery-platform#917]
I notice that ontology annotation relate to different properties of the data set manager's Attribute model.

For example, the term _NCBITaxon#10090_ describes `value`, i.e. _Mus musculus_:

```
  id  | node_id |      type       |      subtype      |                                 | value_unit | value_accession                           | value_source 
------+---------+-----------------+-------------------+---------------------------------+------------+-------------------------------------------+--------------
    1 |       1 | Characteristics | organism          | Mus musculus                    |            | 10090                                     | NCBITaxon
...
 1564 |     603 | Factor Value    | cell type         | Common myeloid progenitor (CMP) |            | http://purl.obolibrary.org/obo/CL_0000049 | CL
```

But in the following example the term _http://purl.obolibrary.org/obo/UO_0000035_ describes `value_unit`, i.e. _mo_:

```
  id  | node_id |     type     |      subtype       |              value              | value_unit |              value_accession              | value_source 
------+---------+--------------+--------------------+---------------------------------+------------+-------------------------------------------+--------------
 1205 |     378 | Factor Value | age                | 22-24                           | mo         | http://purl.obolibrary.org/obo/UO_0000035 | UO
```

My question is: is this intended? How does the Refinery-Platform know which property is being described by the ontology term?

**Comment by ngehlenborg** [comment]
If there is not value_unit entry then the value_accession refers to the value.

**Comment by flekschas** [comment]
I've had a look at Stem Cell Commons data and the assumption is true. I'd filter annotations out that only describe the unit of a value as they don't really help exploring. For example, knowing that a study sample is annotated with _age_ but not seeing the actual age (in the exploration interface) doesn't really help.

**Comment by ngehlenborg** [comment]
I agree!

**Comment by flekschas** [comment]
Has been addressed with 44921bc0458c85bee159a3b17cb8d90a6604e12a
